src/components/table/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC } from 'react';
+import {
+  Table as ChakraTable,
+  TableCellProps as ChakraTableCellProps,
+  TableColumnHeaderProps as ChakraTableColumnHeaderProps,
+} from '@chakra-ui/react';
+export type {
+  TableBodyProps,
+  TableCaptionProps,
+  TableColumnGroupProps,
+  TableColumnProps,
+  TableFooterProps,
+  TableHeaderProps,
+  TableRootProps,
+  TableRowProps,
+  TableScrollAreaProps,
+} from '@chakra-ui/react';
+
+const {
+  Body,
+  Caption,
+  Column,
+  ColumnGroup,
+  Footer,
+  Header,
+  Root,
+  Row,
+  ScrollArea,
+} = ChakraTable;
+
+Body.displayName = 'Table.Body';
+Caption.displayName = 'Table.Caption';
+Column.displayName = 'Table.Column';
+ColumnGroup.displayName = 'Table.ColumnGroup';
+Footer.displayName = 'Table.Footer';
+Header.displayName = 'Table.Header';
+Root.displayName = 'Table.Root';
+Row.displayName = 'Table.Row';
+ScrollArea.displayName = 'Table.ScrollArea';
+
+export type TableCellProps = ChakraTableCellProps & { isNumeric?: boolean };
+const Cell: FC<TableCellProps> = ({ isNumeric, children, ...props }) => (
+  <ChakraTable.Cell
+    {...props}
+    {...{ ...(isNumeric ? { 'data-is-numeric': true } : {}) }}
+  >
+    {children}
+  </ChakraTable.Cell>
+);
+Cell.displayName = 'Table.Cell';
+
+export type TableColumnHeaderProps = ChakraTableColumnHeaderProps & {
+  isNumeric?: boolean;
+};
+const ColumnHeader: FC<TableCellProps> = ({
+  isNumeric,
+  children,
+  ...props
+}) => (
+  <ChakraTable.ColumnHeader
+    {...props}
+    {...{ ...(isNumeric ? { 'data-is-numeric': true } : {}) }}
+  >
+    {children}
+  </ChakraTable.ColumnHeader>
+);
+ColumnHeader.displayName = 'Table.ColumnHeader';
```

Comment:
`isNumeric` prop was removed. We can either add this for compatibility OR apply `textAlign="end"` prop instead

Reply:
I would make it a breaking change and align over backwards compatibility, thinking there will not be 100s of cases where we need that

Reply:
That's another path we can take for sure. I did it mainly to showcase [custom conditions in the theme](https://github.com/smg-automotive/components-pkg/pull/1206/files/1c7ba6481c30401b0976b8f1f1356b103d419d16..f36a02fc5ae9edee2ffcbc6cc3e4db993a2aa130#diff-c77f4d9f3e55c13aabbb5549cbbcf19866be152e265c7774455b1e45e5f2ecf4R4)

Reply:
I ended up removing the custom `isNumeric` prop. I added [a migration guide](https://github.com/smg-automotive/components-pkg/blob/4d881587a2f2db107067fde980991a7702ed16c4/migration-guide.md) where we can reference all breaking changes